scripts/contrib/completion/bash/netctl

```diff
@@ -1,11 +1,19 @@
-#! /bin/bash
+#!/bin/bash
 
 GLOBAL_COMMANDS="\
+    version\
     group\
+    endpoint\
+    netprofile\
     network\
     tenant\
     policy\
+    external-contracts\
     global\
+    aci-gw\
+    bgp\
+    app-profile\
+    service\
     help"
```

Comment:
That sounds good.  You weren't included in that "we" FYI.  That's our problem  😄